deploy/general-deploy/kubeturbo-with-kubeconfig.yaml

```diff
@@ -7,25 +7,25 @@ metadata:
 spec:
   containers:
   - name: kubeturbo
-    image: vmturbo/kubeturbo:latest
+    image: vmturbo/kubeturbo:<VERSION>
     command:
       - /bin/kubeturbo
     args:
       - --v=2
-      - --kubeconfig=<PATH_TO_KUBECONFIG>
+      - --kubeconfig=<PATH_TO_KUBECONFIG_IN_CONTAINER>
       - --turboconfig=/etc/kubeturbo/config
     volumeMounts:
     - name: turbo-config
       mountPath: /etc/kubeturbo
       readOnly: true
     - name: kubeconfig-dir
-      mountPath: <DIRECTORY_CONTAINS_KUBECONFIG>
+      mountPath: <DIRECTORY_CONTAINS_KUBECONFIG_IN_CONTAINER>
```

Comment:
In these yaml files, I think we can give user an runnable example, or almost runnable example:
    set **kubeconfig** and **mouthPath** with some default settings.